### PR TITLE
Milestone 2

### LINKED ISSUE #6: Form for Time Input
- [x] Create 4 inputs for [Days, Hours, Minutes, Seconds]
- [x] Hours value ( 0 - 23 )
- [x] Minutes & Seconds value ( 0 - 59 )
- [x] When the input is entered the total calculation to `seconds` should be shown 
- [ ] Create the Component test

### PR DESCRIPTION
**Pull request by PremanandChowdhury**
**Feature Description:**

1. Have the input for `days : hrs : min : sec`
2. Fetch the input data
3. Display the total inputs into seconds

### CODE REVIEW

File: src/components/countDown/CountDown.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    const id = e.target.id
+    const inputVal = e.target.value
+
+    if (id === 'day') {
+      setDay(inputVal)
+    } else if (id === 'hr' && inputVal < 24 && inputVal >= 0) {
+      setHr(inputVal)
+    } else if (id === 'min' && inputVal < 60 && inputVal >= 0) {
+      setMin(inputVal)
+    } else if (id === 'sec' && inputVal < 60 && inputVal >= 0) {
```
Comment (swarajpure) [inline src/components/countDown/CountDown.js:23]: I suppose the `<` conditions could be `<=`
I'm not able to enter 24 hours 🤔

Comment (harshith-venkatesh) [reply src/components/countDown/CountDown.js:23]: But 24 hours is a day right, are we allowing users to write n hours too?

Comment (PremanandChowdhury) [reply src/components/countDown/CountDown.js:23]: In `Timer` we have numbers from `0 - 23` only for Hours.
I have checked in my phone.
And for min and sec we have `0 - 59`

Comment (PremanandChowdhury) [reply src/components/countDown/CountDown.js:23]: > I suppose the `<` conditions could be `<=`
> I'm not able to enter 24 hours 🤔

In Timer we have numbers from `0 - 23` only for Hours
And for min and sec we have `0 - 59`
I have checked in my phone

Comment (PremanandChowdhury) [reply src/components/countDown/CountDown.js:23]: > But 24 hours is a day right, are we allowing users to write n hours too?

Yes, we are allowing the user to enter no. of hours.

**Pull request merged**